### PR TITLE
1_Mongte/create todo-app

### PR DESCRIPTION
**Pull request by AtlasSchmidt**
- add: add new_button
- add: input - add &cancel button

### CODE REVIEW

**Comment by oddcommits** [comment]
resolve conflicts

**Comment by oddcommits** [comment]
resolve conflicts

File: README.md
Comment (tedzchow) [inline README.md:1]: don't put changes on this file just except contributor list

File: mongte/todo/postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    plugins: [
+      require('tailwindcss'),
+      require('autoprefixer'),
+    ],
+  };
```
Comment (tedzchow) [inline mongte/todo/postcss.config.js:6]: empty line at the end is needed

File: mongte/todo/src/App.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (tedzchow) [inline mongte/todo/src/App.css:41]: [same as here](https://github.com/oddcommits/git-practice/pull/61/files#diff-81a4f28e5e10266263d7e9e38af0ec4b2cfc0abe22977687cca7de55191f43b5)

File: mongte/todo/src/input.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (tedzchow) [inline mongte/todo/src/input.css:3]: [same as here](https://github.com/oddcommits/git-practice/pull/61/files#diff-81a4f28e5e10266263d7e9e38af0ec4b2cfc0abe22977687cca7de55191f43b5)

File: mongte/todo/src/tailwind.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (tedzchow) [inline mongte/todo/src/tailwind.css:3]: [same as here](https://github.com/oddcommits/git-practice/pull/61/files#diff-81a4f28e5e10266263d7e9e38af0ec4b2cfc0abe22977687cca7de55191f43b5)

File: mongte/todo/tailwind.config.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{html,js}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
```
Comment (tedzchow) [inline mongte/todo/tailwind.config.js:8]: [same as here](https://github.com/oddcommits/git-practice/pull/61/files#diff-81a4f28e5e10266263d7e9e38af0ec4b2cfc0abe22977687cca7de55191f43b5)

**Review by oddcommits — CHANGES_REQUESTED** [review]
resolve conflicts as well

**Pull request closed without merging**